app.js

```diff
@@ -129,29 +188,59 @@ function header(message) {
  * @param {puppeteer.Browser} browser
  * @returns {Promise<boolean>} Are there any drops?
  */
-async function hasValorantDrop(browser) {
+async function hasDrops(browser) {
```

Comment:
Parametrize this function by the game name

Reply:
ok